app/controllers/api_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ApiController < ActionController::API
```

Comment:
This looks very familiar. I wonder if we could create a cul RESTful gem with some of these defaults.

Reply:
Yeah, there are a bunch of things we're doing in Hyacinth 3 that are also useful here.  Making a gem may be beneficial.